docs/plugins/pluggable_rule_engine.md

```diff
@@ -98,7 +98,10 @@ def pep_resource_close_pre(rule_args, callback, rei):
     callback.writeLine("serverLog", "XXXX - core.py")
 
     # Instruct the REPF to let the next rule engine plugin handle this PEP.
-    # If we remove the line below, the PEP in core.re will not execute.
+    # If we remove the line below, the PEP in core.re will not execute. If an
```

Comment:
"the PEP" -> "neither matching PEP"

Reply:
"neither matching PEP in core.re will not execute." reads very weird to me.

Should "not" be removed?

Reply:
correct - take out the "not"

Reply:
done